### ISSUE TITLE
Halp?

### ISSUE THREAD

**Issue by Gloomshroud** [Hotride/OrionUO#2]
Any chance we could get an English translation? I'd be willing to do a pull request, but I have no idea what has been done, what is being worked on, or what issues are currently prioritized. I could go through the code base, pick something at random, and start cracking...but that's hardly productive. Russian is all well and good, but I don't speak it. I speak code though. This project is amazing, and I want to help, but I'd have to learn Russian first, and I just don't have the time. :(

**Comment by AimedNuu** [comment]
Hi, you should visit our bugtracker http://bt.orion-client.online 
It has roadmap and tickets which we've been working on. It doesn't include tickets with work on assistant program though.
I promised to translate comments into English and Timur would do commit messages in English if at least 1 active developer (except for us ) would get involved.
You can also come to discord ( https://discord.gg/4K5Rw ) or forums( http://forum.orion-client.online/ )
We're 24/7 on Discord and have a dev channel there.

**Comment by DarkLotus** [comment]
I really dont think we need your effort wasted on translating comments / commit messages @AimedNuu  Rather you working on code.

Commit messages translate with google translate just fine.
The actual code is well written and easy to follow as well without being able to read russian comments, after all class names and variables, functions are all in english so its really no problem.

If your proficient at C++ I dont think anyone would have an  issue getting started, as demonstrated by me even with only some C knowledge and no real experience with C++  i was able to get into the code and track down some issues.

I've also had no problem throwing in some of my own windows messaging hooks for my own easyuo style project for orion.

TL;DR Keep focused on the coding guys, if anyone really wants to help they can, english comments or not, the code explains it self.
